taxi/migrations/0003_alter_car_drivers.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.1 on 2024-11-12 11:59
+
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('taxi', '0002_alter_driver_license_number'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='car',
+            name='drivers',
+            field=models.ManyToManyField(blank=True, null=True, related_name='cars', to=settings.AUTH_USER_MODEL),
```

Comment:
The `null=True` parameter is not applicable to `ManyToManyField` and should be removed. `ManyToManyField` does not support `null=True` because the relationship is managed through a separate table.